include/irr/asset/ISpecializedShader.h

```diff
@@ -12,6 +12,16 @@ namespace irr
 namespace asset
 {
 
+//! Interface class for Specialized Shaders
+/*
+	Specialized shaders are shaders prepared to be attached at pipeline
+	process creation. SpecializedShader consists of unspecialzed Shader
+	containing glsl code, and creation information parameters such as
+	entry point to a shader or stage of a shader.
+
```

Comment:
plesase cover specialization constants

Reply:
adjusted in following commit

Reply:
you haven't covered what a specialization constant is

Reply:
corrected in following commit